packages/amazonq/test/e2e/lsp/lspInstallerUtil.ts

```diff
@@ -201,12 +202,6 @@ export function createLspInstallerTests({
                         id: lspConfig.id,
                         manifestLocation: 'remote',
                         languageServerSetupStage: 'getManifest',
-                        result: 'Failed',
```

Comment:
this was broken in a different commit. Now the manifest download falls back to the local cache if an etag is found but it still reports the telemetry metric as "remote"

Technically this is a minor bug with reporting telemetry, since it should have been reported as a cache